pefile.py

```diff
@@ -2697,7 +2702,6 @@ def get_symbol_name(symbol, string_table, string_table_size):
                             name = s
                         else:
                             end = string_table[offset:].find(0)
-                            print(offset, end)
```

Comment:
My script was producing a lot of output in the most recent version of _pefile_ and I identified this as the line producing it.